### DISCUSSION TITLE
Sunbird RC Install

### DISCUSSION THREAD

**Discussion by tanvirquader** [Sunbird-RC/community#191] [General]
Hi, I am Tanvir from Bangladesh.
We are interested in exploring Sunbird RC, how can I get the deployment architecture?
Thanks in advance.

**Comment by gamemaker1** [comment]
Hi @tanvirquader,

If you wish to try out the Registry locally you could use the Registry CLI following [this guide](https://docs.sunbirdrc.dev/developer-documentation). For pointers on deployment and usage, I think @dileepbapat would be able to help out.

Hope this helps

**Comment by parthlawate** [comment]
Also do go through the docs site https://docs.sunbirdrc.dev/  overall . this should answer most of your questions

**Comment by parthlawate** [comment]
@tanvirquader Wanted to know if you need any further help on this ?
